indico/modules/events/registration/models/registrations.py

```diff
@@ -755,9 +761,17 @@ def generate_ticket_google_wallet_url(self):
         if not self.registration_form.ticket_google_wallet:
             return None
         gwm = GoogleWalletManager(self.event)
-        if gwm.configured:
+        if gwm.is_configured:
             return gwm.get_ticket_link(self)
 
+    def generate_ticket_apple_wallet(self):
+        """Return link to Google Wallet ticket display."""
```

Comment:
Wrong docstring

```suggestion
        """Download ticket in Passbook / Apple Wallet format."""
```

or something like that